v1.1.1.md

```diff
@@ -0,0 +1,101 @@
+# Harvester v1.1.1 Release Note
+
+This release follows Harvester v1.1.0, the documentation is available at https://docs.harvesterhci.io/v1.1
+
+# Downloads
+
+## ISO
+:cd: [harvester-v1.1.1-amd64.iso](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-amd64.iso)
+
+## Additional Artifacts
+:file_folder: [harvester-v1.1.1-initrd-amd64](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-initrd-amd64)
+:file_folder: [harvester-v1.1.1-vmlinuz-amd64](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-vmlinuz-amd64)
+:file_folder: [harvester-v1.1.1-rootfs-amd64.squashfs](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-rootfs-amd64.squashfs)
+:white_check_mark: [harvester-v1.1.1-amd64.sha512](https://releases.rancher.com/harvester/v1.1.1/harvester-v1.1.1-amd64.sha512)
+:memo:  [harvester-v1.1.1-version.yaml](https://releases.rancher.com/harvester/v1.1.1/version.yaml)
+
+# Install
+Harvester can be installed with the [ISO](https://docs.harvesterhci.io/v1.1/install/iso-install/) image or via [PXE](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/) boot.
+
+> **Note**
+>
+> **Configuration change**: 
+> Starting from v1.1.0, the [`install.networks`](https://docs.harvesterhci.io/v1.0/install/harvester-configuration#installnetworks) field is deprecated and replaced by the [`install.management_interface`](https://docs.harvesterhci.io/v1.1/install/harvester-configuration#installmanagement_interface) field. Please check the example file [here](https://github.com/harvester/ipxe-examples/blob/55dab7661a5c14fdf7a8da52b35e40c349f0d03c/general/config-create.yaml#L13-L20).
+ 
+
+# Highlights
+* Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface [[#3045](https://github.com/harvester/harvester/issues/3045)]
+* Host IP is inconsistent [[#2962](https://github.com/harvester/harvester/issues/2962)]
+* Installer crash after input hostname [[#2937](https://github.com/harvester/harvester/issues/2937)]
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
```

Comment:
Remove two indents here. 

Move the two statements to the top of the section, and briefly explain why skipping version v1.1.0. Those two statements are the more critical part of the `Upgrade` section. They need to be front and center.